cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/ExecutionProposal.java

```diff
@@ -106,10 +108,12 @@ public boolean isInterBrokerMovementCompleted(Node[] currentOrderedReplicas) {
    * In that case, we treat it as aborted as well.
    *
    * @param currentOrderedReplicas Current ordered replica list from the cluster.
+   * @param currentInSyncReplicas Current in-sync replica list from the cluster.
    * @return True if aborted, false otherwise.
    */
-  public boolean isInterBrokerMovementAborted(Node[] currentOrderedReplicas) {
-    return isInterBrokerMovementCompleted(currentOrderedReplicas) || brokerOrderMatched(currentOrderedReplicas, _oldReplicas);
+  public boolean isInterBrokerMovementAborted(Node[] currentOrderedReplicas, Node [] currentInSyncReplicas) {
```

Comment:
Nit: [Also applies to `isInterBrokerMovementCompleted`] Should we pass a `PartitionInfo` rather than array of nodes?